include/aero/actuator/ActuatorFunctorsFAST.h

```diff
@@ -164,6 +164,14 @@ using ActFastSpreadForceWhProjection = GenericLoopOverCoarseSearchResults<
   ActuatorBulkFAST,
   ActFastSpreadForceWhProjInnerLoop>;
 
+using ActFastComputeThrustTurbineSearch = GenericLoopOverCoarseTurbineSearchResults<
+  ActuatorBulkFAST,
+  ActFastComputeThrustInnerLoop>;
+
+using ActFastSpreadForceWhProjectionTurbineSearch = GenericLoopOverCoarseTurbineSearchResults<
```

Comment:
In this function in amr-wind there is a distance check that is done prior to computing the exponentials.  In nalu-wind we used the search to filter this, but we probably want to add in the distance check here since we will now have the entire box surrounding the turbine.

Reply:
See my comment below